StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/VerticalPaymentMethodListView.swift

```diff
@@ -42,14 +53,26 @@ class VerticalPaymentMethodListView: UIView {
         var views = [UIView]()
         // Saved payment methods:
         if let savedPaymentMethod {
-            // TOOD(porter) Pass in correct `accessoryType`
-            let accessoryButton = RowButton.RightAccessoryButton(accessoryType: .viewMore, appearance: appearance, didTap: didTapAccessoryButton)
-            let savedPaymentMethodButton = RowButton.makeForSavedPaymentMethod(paymentMethod: savedPaymentMethod, appearance: appearance, rightAccessoryView: accessoryButton) { [weak self] in
+            var accessoryButton: RowButton.RightAccessoryButton?
+            if let rightAccessoryType {
+                accessoryButton = RowButton.RightAccessoryButton(accessoryType: rightAccessoryType, appearance: appearance, didTap: didTapAccessoryButton)
+            }
+            let savedPaymentMethodButton = RowButton.makeForSavedPaymentMethod(paymentMethod: savedPaymentMethod,
+                                                                               appearance: appearance,
+                                                                               rightAccessoryView: accessoryButton)
+            { [weak self] in
                 self?.didTap(rowButton: $0, selection: .saved(paymentMethod: savedPaymentMethod))
             }
-            // Make the saved PM the default selected
-            savedPaymentMethodButton.isSelected = true
-            currentSelection = .saved(paymentMethod: savedPaymentMethod)
+
+            // Select saved payment method button if current selection is a saved payment method or if current selection is nil
+            if case .saved(let savedPaymentMethod) = currentSelection {
```

Comment:
Gahh I also implemented this.  Can we rename to `initialSelection` (what I named this) to make my merge conflicts a bit easier?